plugins/azure_sentinel/help.md

```diff
@@ -0,0 +1,700 @@
+|orderBy|string|None|False|Field to sort results by|None|properties/createdTimeUtc desc|
+|resourceGroupName|string|None|True|The name of the resource group within the user's subscription|None|resourcegrup1|
+|subscriptionId|string|None|True|Azure subscription ID|None|0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe|
+|workspaceName|string|None|True|The name of the workspace|None|workspace23|
+
+Example input:
+
+```
+{
+  "orderBy": "properties/createdTimeUtc desc",
+  "resourceGroupName": "resourcegrup1",
+  "subscriptionId": "0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe",
+  "workspaceName": "workspace23"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|incidents|[]Incident|False|List of incidents objects|
+
+Example output:
+```
+{
+  "incidents":[
+    {
+      "id":"/subscriptions/eeee-aaaa-aaa-eeee-eeeeee/resourceGroups/integrationLab/providers/Microsoft.OperationalInsights/workspaces/sentinel/providers/Microsoft.SecurityInsights/Incidents/1407186457",
+      "name":"1407186457",
+      "etag":"\"8701a4d2-0000-0100-0000-621666620000\"",
+      "type":"Microsoft.SecurityInsights/Incidents",
+      "properties":{
+        "title":"Test incident",
+        "description":"This is a demo incident",
+        "severity":"Low",
+        "status":"Closed",
+        "classification":"FalsePositive",
+        "classificationReason":"IncorrectAlertLogic",
+        "classificationComment":"Not a malicious activity",
+        "labels":[],
+        "firstActivityTimeUtc":"2019-01-01T13:00:30Z",
+        "lastActivityTimeUtc":"2019-01-01T13:05:30Z",
+        "lastModifiedTimeUtc":"2022-02-23T16:52:50.1945053Z",
+        "createdTimeUtc":"2022-02-23T16:52:50.1945053Z",
+        "incidentNumber":3,
+        "additionalData":{
+          "alertsCount":0,
+          "bookmarksCount":0,
+          "commentsCount":0,
+          "alertProductNames":[]
+        },
+        "relatedAnalyticRuleIds":[],
+        "incidentUrl":"https://portal.azure.com/#asset/Microsoft_Azure_Security_Insights/Incident/subscriptions/eeeee-eeee-eee-eeee-df965e27aefe/resourceGroups/integrationLab/providers/Microsoft.OperationalInsights/workspaces/sentinel/providers/Microsoft.SecurityInsights/Incidents/1407186457"
+      }
+    }
+  ]
+}
+```
+#### Create or Update Incident
+
+This action creates or updates an incident.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|incidentId|string|None|True|Incident ID|None|incident-14071867|
+|properties|IncidentProperties|None|True|Incident properties object|None|{'status': 'Closed'}|
+|resourceGroupName|string|None|True|The name of the resource group within the user's subscription|None|resourcegrup1|
+|subscriptionId|string|None|True|Azure subscription ID|None|aaaef455-a780-44ca-9e51-aaafffeeea3a|
+|workspaceName|string|None|True|The name of the workspace|None|workspace23|
+
+Example input:
+
+```
```

Comment:
Kindly add the input. Seems like the properties in the input is a complex object and seems like help.md did not render correctly. Can you please check.